views/networking/load_balancer/create.erb

```diff
@@ -54,8 +54,8 @@
                 ) %>
               </div>
             </div>
-            <div class="mt-6 grid grid-cols-1 gap-x-6 gap-y-8 sm:grid-cols-6">
-              <div class="col-span-6 md:col-span-2 xl:col-span-1">
+            <div class="mt-6 grid grid-cols-2 gap-x-6 gap-y-8 sm:grid-cols-6">
```

Comment:
In this PR, I tried to locate where we remove health check input fields. However, I realized that this removal is in the API PR. Should we move that commit to this PR?